config/configuration_keys.example.json

```diff
@@ -1,13 +0,0 @@
-{
```

Comment:
Why is this file removed?

Reply:
This file is not removed but is renamed as "configuration_keys.json". because this file is imported as "configuration_keys.json" which is why it was throwing error "module not found".

Reply:
I am not seeing a renamed file. It is just removed. [Head of your branch](https://github.com/PSUCompBio/nsfcareer-react-website/tree/createHomeScreenUI/config).

But that does not matter because it should not be removed or renamed. The naming is intentional. This is an example config file, named intentionally as "example". Not designed to be used.

User is supposed to create their own "configurations_keys.json" which is not tracked on GitHub, mentioned [here](https://github.com/PSUCompBio/nsfcareer-react-website#important-do-not-upload-access-keys-to-web-eg-github-google-drive).